### ISSUE TITLE
Some sort of progress indicator

### ISSUE THREAD

**Issue by codeforger-wp** [jamesmorrison/apollo#11]

**Comment by codeforger-wp** [comment]
Any sort of ticker on the last line to show that the process has not frozen

**Comment by jamesmorrison** [comment]
>Any sort of ticker on the last line to show that the process has not frozen

During provision @codeforger-wp ?

**Comment by codeforger-wp** [comment]
yes, if you can get something like a pipe spinner (it outputs `|` `/` `-` `\` in sequence).

then while that is spinning people will not get worried the provision has stalled
